ipaserver/install/ipa_kra_install.py

```diff
@@ -83,33 +83,13 @@ def validate_options(self, needs_root=True):
     @classmethod
     def get_command_class(cls, options, args):
         if options.uninstall:
-            return KRAUninstaller
+            raise RuntimeError(
```

Comment:
`ipa-kra-install --uninstall` prints a traceback instead of just showing the error message.